src/dashboards/components/dashboard_index/DashboardCard.tsx

```diff
@@ -231,11 +231,13 @@ class DashboardCard extends PureComponent<Props> {
   private handleDeleteDashboard = async () => {
     const {id, name, onDeleteDashboard} = this.props
     onDeleteDashboard(id, name)
-    try {
-      await deletePinnedItemByParam(id)
-      this.props.sendNotification(pinnedItemSuccess('task', 'deleted'))
-    } catch (error) {
-      this.props.sendNotification(pinnedItemFailure(error.message, 'delete'))
+    if (isFlagEnabled('pinnedItems') && CLOUD && this.props.isPinned) {
```

Comment:
Why `CLOUD` only? Doesn't exist for OSS?

Reply:
correct, it's a cloud-only feature because it uses a backend nodejs service that isn't available in OSS installs